scripts/archlinux/solidity-git/PKGBUILD

```diff
@@ -0,0 +1,65 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity-git/
+
+pkgname=solidity-git
+pkgver=0.4.3.git.20161020.9d30450
```

Comment:
Yes, and as discussed in #1152 it's not sufficient. I need some counter of commits since the last tag as per arch linux package build requirements. since this is not available due to the way you tag releases on a separate branch I use a date. does not look nice, but was the best I could come up with.

The easiest way would be to simply merge release into develop ;-)